tla/tlc.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+#
+# Python TLC wrapper script for the CCF project
+# Goals:
+# - No dependencies, no venv, no pip install
+# - Set sensible defaults with an eye on performance
+# - Capture useful switches for CI, debugging
+# - Expose specification configuration through CLI
+# - Provide a useful --help, and basic sanity checks
+
+import os
+import argparse
+import pprint
+import pathlib
+
+DEFAULT_JVM_ARGS = [
+    "-XX:+UseParallelGC",  # Use parallel garbage collection, for performance
+]
+
+
+DEFAULT_CLASSPATH_ARGS = ["-cp", "tla2tools.jar:CommunityModules-deps.jar"]
+
+
+def cli():
+    parser = argparse.ArgumentParser(
+        description="TLC model checker wrapper for the CCF project"
+    )
+
+    # Common options for all commands
+    parser.add_argument(
+        "-v",
+        action="store_true",
+        help="Print out command and environment before running",
```

Comment:
can we have a print without running too? might be useful

Reply:
Sure, you get to pick a name for it :)

Reply:
what an honor, I pick `-n` for consistency with set https://linuxcommand.org/lc3_man_pages/seth.html